specification/maps/data-plane/DataRegistry/stable/2022-12-01/dataregistry.json

```diff
@@ -0,0 +1,712 @@
+          }
+        }
+      },
+      "delete": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThis API allows the caller to delete a previously created data registry.\n\nYou can use this API in a scenario like removing geofences previously uploaded using the [Data Create API](https://docs.microsoft.com/en-us/rest/api/maps/data-v2/upload-preview) for use in our [Azure Maps Geofencing Service](https://docs.microsoft.com/en-us/rest/api/maps/spatial). You can also use this API to delete old/unused created udid.\n\n## Submit Delete Request\n\nTo delete your content you will issue a `DELETE` request where the path will contain the `udid` of the data to delete.\n\nFor example, to delete a collection of geofences previously created using the DataRegistry Create API, set the `udid` parameter in the path to the `udid` of the data received previously in the DataRegistry Create API response.\n\n## Delete Data Response\n\nThe Data Delete API returns a HTTP `204 No Content` response with an empty body, if the data resource was deleted successfully.\n\nA HTTP `400 Bad Request` error response will be returned if the data resource with the passed-in `udid` is not found.",
+        "operationId": "DataRegistries_Delete",
+        "x-ms-examples": {
+          "Delete previously uploaded GeoJSON data containing geometries that represent a collection of geofences": {
+            "$ref": "./examples/DataRegistries_Delete.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "../../../Common/preview/1.0/common.json#/parameters/ClientId"
+          },
+          {
+            "$ref": "#/parameters/ApiVersion"
+          },
+          {
+            "$ref": "#/parameters/UniqueDataId"
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Data delete request completed successfully. The reference to your content was deleted in Azure Maps Service."
+          },
+          "default": {
+            "description": "An unexpected error occurred.",
+            "schema": {
+              "$ref": "../../../../../common-types/data-plane/v1/types.json#/definitions/ErrorResponse"
+            },
+            "x-ms-error-response": true
+          }
+        }
+      }
+    },
+    "/mapData/{udid}/metadata": {
+      "get": {
+        "description": "**Applies to:** see pricing [tiers](https://aka.ms/AzureMapsPricingTier).\n\nThis API allows the caller to fetch the metadata of a given udid created previously using the [DataRegistry Create API](https://docs.microsoft.com/en-us/rest/api/maps/data-v2/upload-preview).\n\n## Submit Get Metadata Request\n\nTo fetch the metadata of a given udid you will issue a `GET` request with no additional parameters.\n\n## Get Metadata Response\n\nThe Data Metadata API returns the complete list of all data in `json` format.",
+        "operationId": "DataRegistries_GetMetadata",
+        "x-ms-examples": {
```

Comment:
Consider adding another example when the `isValid` is false so that customer know how to debug.

Reply:
Added